docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -1,175 +1,268 @@
+stream before restoring it. To prevent the accidental recreation of the index or
+data stream, we recommend you temporarily stop all indexing until the restore
+operation is complete.
+
+// Only display warning in 7.x.
+// In 7.x, `action.destructive_requires_name` defaults to false.
+ifeval::[ "{major-version}" == "7.x" ]
+WARNING: Don't use the <<indices-delete-index,delete index API>> to target the
+`*` or `.*` wildcard expressions. If you use {es}'s security features, this will
+delete system indices required for authentication. To delete all regular
+indices, use `*,-.*`.
+endif::[]
 
 [source,console]
------------------------------------
-POST /_snapshot/my_backup/snapshot_1/_restore
+----
+# Delete an index
+DELETE my-index
+
+# Delete a data stream
+DELETE _data_stream/logs-my_app-default
+
+# Restore the index and data stream from a snapshot
+POST _snapshot/my_repository/my_snapshot_2099.05.06/_restore 
 {
-  "indices": "data_stream_1,index_1,index_2",
-  "ignore_unavailable": true,
-  "include_global_state": false,              <1>
-  "rename_pattern": "index_(.+)",
-  "rename_replacement": "restored_index_$1",
-  "include_aliases": false
+  "indices": "my-index,logs-my_app-default"
 }
------------------------------------
-// TEST[continued]
-
-<1> By default, `include_global_state` is `false`, meaning the snapshot's
-cluster state and feature states are not restored.
-+
-If `true` then the restore operation merges the legacy index templates in your
-cluster with the templates contained in the snapshot, replacing any existing
-ones whose name matches one in the snapshot. It completely removes all
-persistent settings, non-legacy index templates, ingest pipelines and
-{ilm-init} lifecycle policies that exist in your cluster and replaces them with
-the corresponding items from the snapshot.
-
-The restore operation must be performed on a functioning cluster. However, an
-existing index can only be restored if it's <<indices-close,closed>> and
-has the same number of shards as the index in the snapshot. The restore
-operation automatically opens restored indices if they were closed and creates
-new indices if they didn't exist in the cluster.
-
-If a data stream is restored, its backing indices are also restored. The restore
-operation automatically opens restored backing indices if they were closed.
-
-In addition to entire data streams, you can restore only specific backing
-indices from a snapshot. However, restored backing indices are not automatically
-added to any existing data streams. For example, if only the
-`.ds-logs-2099.03.08-000003` backing index is restored from a snapshot, it is
-not automatically added to the existing `logs` data stream.
+----
+// TEST[setup:setup-snapshots]
+// TEST[s/_restore/_restore?wait_for_completion=true/]
 
 [discrete]
-=== Partial restore
+[[close-restore]]
+==== Close and restore
+
+You can restore an existing index if it's closed. However, the index in the
```

Comment:
I believe this maybe is suppose to merge restore data if the index is only closed & not deleted. (aka. restores un-findable records) -- as a comment on why you'd choose one & not the other

Reply:
Unfortunately, this isn't the case. Restoring a closed index overwrites its existing data. It's more or less the same as delete and restore. I updated the copy to note this in 986850c69e5c9bf7a2534ffccd25906d38401276.